### ISSUE TITLE
Secure Invoices: only admins can view

### ISSUE THREAD

**Issue by nutsandbolts** [sliced-invoices/sliced-invoices#20]
(iref 27763)

When the Secure extension is in use, only admins are able to view and create invoices and quotes. We should probably add an option for the admin to choose what user roles are able to access when the Secure extension is active.

**Comment by grantalltodavid** [comment]
I don't think this is a bug... I think we just need to sit down and think about user roles and access and what we want to do.  Up to now we've mostly assumed it would just be admins.

**Comment by nutsandbolts** [comment]
True. I've relabeled as an enhancement.
